src/opencl_bitlocker_fmt_plug.c

```diff
@@ -33,26 +31,22 @@ john_register_one(&fmt_opencl_bitlocker);
 #include "formats.h"
 #include "options.h"
 #include "common-opencl.h"
+#include "bitlocker_common.h"
 
-#define FORMAT_LABEL            "bitlocker-opencl"
-#define FORMAT_NAME             ""
+#define FORMAT_LABEL		    "bitlocker-opencl"
+//#define FORMAT_NAME             ""
 #define ALGORITHM_NAME          "PBKDF2-SHA256 AES OpenCL"
-#define FORMAT_TAG              "$bitlocker$"
+//#define FORMAT_TAG              "$bitlocker$"
 #define FORMAT_TAG_LEN       (sizeof(FORMAT_TAG)-1)
 #define SALT_ALIGN		1
 #define BITLOCKER_JTR_HASH_SIZE 45
-#define BITLOCKER_JTR_HASH_SIZE_CHAR 77
-#define MIN_KEYS_PER_CRYPT  1   /* These will change in init() */
+#define PLAINTEXT_LENGTH 208
```

Comment:
Our core code currently accept a PLAINTEXT_LENGTH of max. 125 (not including the trailing NULL)